src/renderer/components/ReactFlowBox.tsx

```diff
@@ -370,6 +370,14 @@ export const ReactFlowBox = memo(({ wrapperRef, nodeTypes, edgeTypes }: ReactFlo
         [createNode, wrapper, reactFlowInstance, schemata, sendAlert, setEdges, setNodes]
     );
 
+    // TODO: I want to get this to work at some point but for now it needs to not exist
+    // const onEdgeUpdate = useCallback(
```

Comment:
I don't really like commented-out code, but we can keep it if you expect to integrate it relatively quickly.

Reply:
I just don't wanna have to dig through the RF discord again to find this snippet

Reply:
Okay, then let's keep it.